tools/c7n_tencentcloud/c7n_tencentcloud/resources/mysql_backup.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod, isoformat_datetime_str
+import pytz
+
+
+@resources.register("mysql-backup")
```

Comment:
is this actually specific to mysql? afaics this api is the same across db types (mysql, Maria, postgresql, etc).

Reply:
never mind it looks like each db type has a separate api endpoint.